app/adapters/postmark_adapter/outbound.rb

```diff
@@ -179,25 +187,28 @@ def message_email
     private
 
     def broadcasted_message_email
-      headers({ 'message-id': "request/#{msg.request.id}@#{Setting.application_host}" })
+      headers({ 'message-id': "request/#{msg.request.id}@#{ENV.fetch('APPLICATION_HOSTNAME', 'localhost:3000')}" })
       email_subject = I18n.t('adapter.postmark.new_message_email.subject')
-      message_stream = Setting.postmark_broadcasts_stream
+      message_stream = ENV.fetch('POSTMARK_BROADCASTS_STREAM', 'broadcasts')
       attach_files if msg.files.present?
-      mail(to: msg.recipient.email, subject: email_subject, message_stream: message_stream)
+      mail(from: email_from_address(organization: msg.request.organization), to: msg.recipient.email, subject: email_subject,
```

Comment:
I think we could introduce a delegation of message.organization to the message.request in the Message model.

Reply:
Good idea. See 7b385af4